src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -300,10 +298,6 @@ public Action updateAction(final Action action) {
     return existingAction;
   }
 
-  @Transactional(
```

Comment:
you probably did want the `@Transactional` annotation, but you don't need to specify propagation, not read only, timeout etc. etc. - the defaults are fine